### PR TITLE
Full button fix + more

### PR DESCRIPTION
**Pull request by LUModder**
- Removed close button from editor
- Fixed buttons not showing
- Experimental set player color

### CODE REVIEW

File: lua/pac3/core/client/parts/entity.lua [hunk 1]
```diff
@@ -203,6 +204,19 @@ function PART:SetColor(var)
 	self.Colorc.b = var.b
 end
 
+function PART:SetPlayerColor(var)
+	var = var or Vector(255,255,255)
```
Comment (PotcFdk) [inline lua/pac3/core/client/parts/entity.lua]: Shouldn't this use Color()?

Comment (LUModder) [inline lua/pac3/core/client/parts/entity.lua]: SetPlayerColor uses vectors but it should be /255 so

File: lua/pac3/core/client/parts/entity.lua [hunk 2]
```diff
@@ -203,6 +204,19 @@ function PART:SetColor(var)
 	self.Colorc.b = var.b
 end
 
+function PART:SetPlayerColor(var)
+	var = var or Vector(255,255,255)
+	
+	self.PColor = var
+	self.PColorf = Vector(var.r,var.g,var.b)/255
```
Comment (PotcFdk) [inline lua/pac3/core/client/parts/entity.lua]: What if var was set to `Vector(255,255,255)` in line 208? `var.r`, `var.g` and `var.b` would be `nil` here.

File: lua/pac3/core/client/parts/entity.lua [hunk 3]
```diff
@@ -203,6 +204,15 @@ function PART:SetColor(var)
 	self.Colorc.b = var.b
 end
 
+function PART:SetPlayerColor(var)
+	var = var or self.Owner:GetPlayerColor()
+	
+	self.PColor = var
+	self.PColorf = Vector(var.x,var.y,var.z)/255
```
Comment (PotcFdk) [inline lua/pac3/core/client/parts/entity.lua]: If `var` is always expected to be a Vector, wouldn't `self.PColorf = var / 255` do the job?

**Comment by LUModder** [comment]
Can we merge already, I'm tired of getting complaints.

**Comment by suchipi** [comment]
If we're making PRs, why are we making them to Metastruct/pac3 instead of CapsAdmin/pac3? My understanding is that the point of reviving Metastruct/pac3 was just to be able to test things without needing to develop remotely or test locally or make a PR to Caps.

**Pull request merged**